py2puml/inspection/inspectclass.py

```diff
@@ -83,7 +75,37 @@ def inspect_static_attributes(
             uml_attr = UmlAttribute(attr_name, attr_type, static=True)
             definition_attrs.append(uml_attr)
 
-    return definition_attrs
+
+def inspect_methods(
+    definition_methods, class_type,
+):
+    no_dunder = lambda x: not (x[0].startswith('__') or x[0].endswith('__'))
```

Comment:
just excluding magic methods

Reply:
😰 ups. lol I was really hacking together. sure thing. also, I tested this with some other projects and it totally shows more than what we want to see. need to look a lot further into determining declared methods only.